differential-pressure-sensor/differential-pressure-sensor.ino

```diff
@@ -0,0 +1,39 @@
+#define Vs 5.0
```

Comment:
Add a short description (even 1 line) of what this sketch is for. It's a lib for MPXV7002 right?